### PR TITLE
Update pre-train model URL(faster rcnn and mask rcnn)

### PR DESCRIPTION
**Pull request by hhaAndroid**
update pre-train model URL(faster rcnn and mask rcnn)

### CODE REVIEW

**Review by ZwwWayne — APPROVED** [review]

**Review by hellock — CHANGES_REQUESTED** [review]
There is a linting error.

**Comment by ZwwWayne** [comment]
Please fix the linting error and also update the performance of these two new models.

File: configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco-person.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
-_base_ = './faster_rcnn_r50_fpn_1x_coco.py'
+_base_ = './faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py'
```
Comment (ZwwWayne) [inline configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco-person.py:1]: Rename the configs to be `faster_rcnn_r50_fpn_mstrain_1x_coco-person.py`

File: configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco-person-bicycle-car.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
-_base_ = './faster_rcnn_r50_fpn_1x_coco.py'
```
Comment (ZwwWayne) [inline configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco-person-bicycle-car.py:1]: Rename the file to be `faster_rcnn_r50_fpn_1x_coco-person-bicycle-car.py`

**Review by hellock — APPROVED** [review]

**Pull request merged**